platform-sdk/swirlds-metrics-api/src/main/java/com/swirlds/metrics/api/Metrics.java

```diff
@@ -36,7 +36,7 @@ public interface Metrics extends Startable {
 
     String INTERNAL_CATEGORY = "internal";
     String PLATFORM_CATEGORY = "platform";
-    String INFO_CATEGORY = "platform.info";
+    String INFO_CATEGORY = "platform:info";
```

Comment:
Will this change metrics names that are current used?

Reply:
It will not change the name on the Prometheus side; they were already called this after the adapter fixed the name.